pkg/reconciler/taskrun/resources/taskref_test.go

```diff
@@ -1384,6 +1372,18 @@ spec:
     type: array
 `
 
+var remoteTaskYamlWithoutDefaults = `
+metadata:
+  name: simple
+  namespace: default
+spec:
+  steps:
+  - image: something
+  params:
+  - name: foo
+    type: string
```

Comment:
```suggestion

```

It looks like the previous test was using a remote task without defaults set, i.e. no type for the param, and testing that defaults (i.e. type = string) were not set-- let's make sure to continue to test the same functionality

Reply:
thanks, updated